test/fixtures/empty-process.bpmn

```diff
@@ -1,3 +1,2 @@
 <bpmn:definitions xmlns:bpmn="http://www.omg.org/spec/BPMN/20100524/MODEL" xmlns:bpmndi="http://www.omg.org/spec/BPMN/20100524/DI" xmlns:di="http://www.omg.org/spec/DD/20100524/DI" xmlns:dc="http://www.omg.org/spec/DD/20100524/DC" id="Definitions_13fbzpq" targetNamespace="http://bpmn.io/schema/bpmn" exporter="Camunda Modeler" exporterVersion="1.16.2">
-  <bpmn:process id="Process_1oh7y86" isExecutable="true" />
```

Comment:
This changes the diagram to not have an empty process anymore. The scenario we want to cover is that the process does exist but is empty.

Reply:
Issue #46 refers to an "empty definition." Maybe the name can be more elaborative.